src/corehost/cli/test/nativehost/singlefile/singlefile.cpp

```diff
@@ -0,0 +1,4 @@
+#include <pal.h>
+
+SHARED_API const char *DotNetRuntime;
+const char *DotNetRuntime = u8"DotNetRuntime";
```

Comment:
There is a change that this label/signature will be a structure with some info after the label like DAC and DBI build id. For testing this currently would be enough but it may change.